inst/include/population_dynamics/population/population.hpp

```diff
@@ -457,7 +457,7 @@ struct Population : public fims_model_object::FIMSObject<Type> {
                  this->weight_at_age[age];
       } else {
         POPULATION_LOG << "fleet " << fleet_ << " is a survey" << std::endl;
-        index_ = this->fleets[fleet_]->q *
+        index_ = this->fleets[fleet_]->q[0] *
                  this->fleets[fleet_]->selectivity->evaluate(ages[age]) *
```

Comment:
Thinking about selectivity here in a vectored world the selectivity at age is now changing through time potentially. We should probably not worry about these issues for this pull request and just flag dimensionality as an issue to clean up now that parameters are vectors.